### PR TITLE
Static Framework Release Note Input for 1.4.0 blog

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

**Comment by dnkoutso** [comment]
@paulb777 do you want me to just copy this text into my other PR? Obviously I will be amending my other PR to include credits to all of our contributors/supporters for 1.4.0.

File: _posts/Static-Framework.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+### Static Frameworks
+
+Source static frameworks is powerful new feature of CocoaPods 1.4.0. For the first time, it is
+possible to distribute source CocoaPods and require that they be built into a static framework
+when `use_frameworks!` is specified in the Podfile. Because dynamic libraries cannot depend
+upon static libraries, it now becomes possible to distribute CocoaPods that depend upon
+other static library frameworks, including the many static frameworks that are
+released as vendored_frameworks.
```
Comment (dnkoutso) [inline _posts/Static-Framework.markdown]: nit: add backticks around `vendored_frameworks`

File: _posts/Static-Framework.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+not have the necessary relocations to be loaded at runtime and because if multiple
+dynamic libraries depended upon the same static library, there would be multiple
+copies of the static library. The multiple copies is only a code size issue for
+the code itself, but it will lead to functionality issues for any data in the
+static library.
+
+When Apple and Xcode refer to *frameworks*, dynamic is typically implied. However, the lower
+level Apple build tools work correctly with both dynamic and static frameworks.
+
+Historically many binary CocoaPods have been distributed as a vendored_framework
```
Comment (dnkoutso) [inline _posts/Static-Framework.markdown]: same on backticks

**Comment by paulb777** [comment]
@dnkoutso Yes, please copy it into your PR. We can close this PR once you do so and can have any subsequent discussions there.

**Comment by dnkoutso** [comment]
SGTM thanks will do and then close this one.

**Comment by orta** [comment]
👍

**Pull request closed without merging**